cypress/e2e/search.cy.js

```diff
@@ -0,0 +1,233 @@
+import SearchPageObjects from "../pageObjects/searchPage";
+
+const searchPage = new SearchPageObjects();
+const tags = ['@search', '@authentcation', '@common', '@root']
+const devices = ['ipad-2', 'ipad-mini', 'iphone-3', 'iphone-4', 'iphone-5', 'iphone-6', 'iphone-6+', 'iphone-7', 'iphone-8', 'iphone-xr', 'iphone-se2', 'macbook-11', 'macbook-13', 'macbook-15', 'macbook-16', 'samsung-note9', 'samsung-s10']
+const filterSearch = ['Last name A-Z', 'Last name Z-A', 'Best match']
+const numberOfResults = ['40', '20', '12']
+
+describe('Search Page',{tags: tags}, ()=> {
+    beforeEach(()=> {
+        cy.login()
+    })
+    it('should search for a property, person and tenure', {'tags': '@SmokeTest'}, ()=> {
+        searchPage.visit();
+        //property
+        const searchTerm = "Avenue"
+        searchPage.searchContainer().type(searchTerm);
+        searchPage.searchButton().click();
+
+        searchPage.searchResults().contains(searchTerm.replace(/\*/g, ""), { matchCase: false });
```

Comment:
I think using Regex, needlessly overcomplicates this expression. :thinking: 
Could use `replaceAll` and the intention would be clearer:

![image](https://github.com/LBHackney-IT/mtfh-tl-e2e-tests/assets/43747286/a0fe46c0-ed5e-4da3-ad3b-95eabbd68583)